docs/data-sources/space.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spacelift_space Data Source - terraform-provider-spacelift"
+subcategory: ""
+description: |-
+  spacelift_space represents a Spacelift space - an inheritable entity holding a collection of customer-defined resources such as stacks, modules, policies, etc. Allows for more granular access control.
+---
+
+# spacelift_space (Data Source)
+
+`spacelift_space` represents a Spacelift **space** - an inheritable entity holding a collection of customer-defined resources such as stacks, modules, policies, etc. Allows for more granular access control.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `space_id` (String) immutable ID (slug) of the space
+
+### Read-Only
+
+- `description` (String) free-form space description for users
+- `id` (String) The ID of this resource.
+- `inherit_entities` (Boolean) indication whether this space inherits entities from the parent space
```

Comment:
indication whether access to this space inherits read access to entities from the parent space.